### PR TITLE
Add RUBY_VERSION feature + pin json_pure/rspec-rubocop

### PR DESCRIPTION
**Pull request by bastelfreak**
this will implement https://github.com/voxpupuli/modulesync_config/issues/198

### CODE REVIEW

File: moduleroot/Gemfile [hunk 1]
```diff
@@ -42,7 +44,7 @@ group <%= group %> do
     gem 'beaker-rspec',  :require => false
   end
 <% else -%>
-  gem '<%= gem['gem'] %>'<%= ", '#{gem['version']}'" if gem['version'] %>, <%= ' ' * (maxlen - gem['length']) %> :require => false<%= ", :git => '#{gem['git']}'" if gem['git'] %><%= ", :branch => '#{gem['branch']}'" if gem['branch'] %><%= ", :platforms => #{gem['platforms']}" if gem['platforms'] %>
+  gem '<%= gem['gem'] %>'<%= ", '#{gem['version']}'" if gem['version'] %>, <%= ' ' * (maxlen - gem['length']) %> :require => false<%= ", :git => '#{gem['git']}'" if gem['git'] %><%= ", :branch => '#{gem['branch']}'" if gem['branch'] %><%= ", :platforms => #{gem['platforms']}" if gem['platforms'] %><%= " if RUBY_VERSION #{gem['ruby-operator']} #{gem['ruby-version']}" if gem['ruby-operator'] && gem['ruby-version'] %>
```
Comment (rnelson0) [inline moduleroot/Gemfile]: As there's a few if's on this line, perhaps the last one's conditions could be wrapped in parens to make it stand out more from the string to the left side.

File: moduleroot/Gemfile [hunk 2]
```diff
@@ -20,13 +20,15 @@ group <%= group %> do
...
-<%              'length'    => gem['gem'].length + (("', '".length if gem['version']) || 0) + gem['version'].to_s.length -%>
+<%              'gem'           => gem['gem'], -%>
+<%              'version'       => gem['version'], -%>
+<%              'platforms'     => gem['platforms'], -%>
+<%              'require'       => gem['require'], -%>
+<%              'git'           => gem['git'], -%>
+<%              'branch'        => gem['branch'], -%>
+<%              'ruby-version'  => gem['ruby-version'], -%>
+<%              'ruby-operator' => gem['ruby-operator'], -%>
+<%              'length'        => gem['gem'].length + (("', '".length if gem['version']) || 0) + gem['version'].to_s.length -%>
```
Comment (igalic) [inline moduleroot/Gemfile]: what's going on here?
(also, monads would be pretty sweet now…)

File: moduleroot/Gemfile (hunk 1 above)
Comment (igalic) [inline moduleroot/Gemfile]: why don't we make this a helper function or two?

Comment (bastelfreak) [inline moduleroot/Gemfile]: +1. but please in a different PR.

File: moduleroot/Gemfile (hunk 2 above)
Comment (bastelfreak) [inline moduleroot/Gemfile]: we make dark black magic to determine the indentation in the Gemfile.

Comment (igalic) [inline moduleroot/Gemfile]: that's what i'm saying we should have a function for

Comment (bastelfreak) [inline moduleroot/Gemfile]: I created https://github.com/voxpupuli/modulesync_config/issues/201 for this

**Pull request merged**